zebra-chain/src/value_balance.rs

```diff
@@ -0,0 +1,248 @@
+//! A type that can hold the Zcash four types of value pools
```

Comment:
Suggestion: minor rewording for clarity

```suggestion
//! A type that can hold the four types of Zcash value pools.
```